selfdrive/modeld/SConscript

```diff
@@ -33,6 +33,8 @@ if arch == "aarch64" or arch == "larch64":
   common_src += thneed_src
   dlsym_offset = get_dlsym_offset()
   lenv['CXXFLAGS'].append("-DUSE_THNEED")
+  # Define _USING_LIBCXX so <stdatomic.h> defers to the <atomic> header. 
+  lenv['CXXFLAGS'].append("-D_USING_LIBCXX")
```

Comment:
why not always define this? this affects anything that includes hardware.h or any of the android libs